tests/integration/models/test_inbox.py

```diff
@@ -90,6 +90,44 @@ def test_message(self):
         assert message.replies == []
         assert isinstance(message.subreddit, Subreddit)
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_message_collapse(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestInbox.test_message_collapse',
+                match_requests_on=['uri', 'method', 'body']):
+            self.reddit.inbox.collapse(list(self.reddit.inbox.messages()))
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_message_uncollapse(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestInbox.test_message_uncollapse',
+                match_requests_on=['uri', 'method', 'body']):
+            self.reddit.inbox.uncollapse(list(self.reddit.inbox.messages()))
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_message_collapse__with_id(self, _):
```

Comment:
Can this method be moved above the previous so that the methods remain in sorted order?